openpype/hosts/blender/plugins/publish/unpause_sync_server.py

```diff
@@ -15,3 +16,10 @@ def process(self, context):
         manager = ModulesManager()
         sync_server_module = manager.modules_by_name["sync_server"]
         sync_server_module.unpause_server()
+
+        # Wait for all started futures to finish
+        for instance in context:
+            for future in instance.data.get("futures", []):
+                while future.running():
```

Comment:
And same here.

Reply:
But for this one, I think keeping the future name helps to understand what it is about.

Reply:
Agreed